### PR TITLE
Fix path passing for pre-commit

### PR DESCRIPTION
**Pull request by kaidokert**
Also fix filename separators on Windows.

b/150775273

### CODE REVIEW

**Comment by kaidokert** [comment]
Intend to submit the first commit of course only, second one is just to show things working

File: .pre-commit-config.yaml
Comment (dahlstrom-g) [inline .pre-commit-config.yaml:1]: Since the PR is already doing two things
maybe also add the `$` on line 40 for consistency.

**Review by dahlstrom-g — APPROVED** [review]
With commit 9a9148fd the `meta-validate` hook works for me locally.

PR-title nit: https://cbea.ms/git-commit/#end

File: .pre-commit-config.yaml
Comment (kaidokert) [reply .pre-commit-config.yaml:1]: done

**Comment by kaidokert** [comment]
> With commit [9a9148f](https://github.com/youtube/cobalt/commit/9a9148fd7f3a030ce32bf71a8b0887a6e85c45cd) the `meta-validate` hook works for me locally.
> 
> PR-title nit: https://cbea.ms/git-commit/#end

done

**Pull request merged**